ui/src/components/forms/FormEditorContainer.js

```diff
@@ -97,6 +99,20 @@ const FormEditorContainer = (props) => {
     setFormManagerElements(editFormData);
   }, [editFormData]);
 
+
+
+  useEffect(() => {
```

Comment:
even though it shows warning igf the date is invalid, its still possible to submit an invalid date. maybe should block saving, to prevent unexpected behaviour.

<img width="596" alt="image" src="https://user-images.githubusercontent.com/24607458/196012909-a749f887-5b62-487f-aac2-88bb2b5e05fb.png">